pkg/apis/skupper/v1alpha1/types.go

```diff
@@ -478,7 +502,8 @@ func (l *Link) setReady(err error) bool {
 }
 
 func (l *Link) isReady() bool {
```

Comment:
could this function be public?

Reply:
Certainly, it can be made public when needed. However thinking  little more about it, from the cli we would probably want to check whether the condition existed at all, i.e. whether it was true or false, as if it is false with an error we would want to report that rather than just waiting indefinitely.